virttest/tests/unattended_install.py

```diff
@@ -399,6 +399,11 @@ def answer_kickstart(self, answer_path):
             raise ValueError("Unexpected installation medium %s" % self.url)
         contents = re.sub(dummy_medium_re, content, contents)
 
+        dummy_virt_module_stream_re = r'\bRH_VIRT_QGA\b'
+        if re.search(dummy_virt_module_stream_re, contents):
+            virt_module_stream = self.params.get("virt_qga", "")
```

Comment:
It is better to keep consistent `rh_virt_qga`

Reply:
@yanan-fu please have a review.